aspnetcore/blazor/hybrid/tutorials/maui-blazor-web-app.md

```diff
@@ -365,19 +392,19 @@ In the `_Imports.razor` file of the `.Shared.Client` RCL, add `@using static Int
 
 The following example demonstrates how to use an interface to call into different implementations across the web app and the native (MAUI) app. The following example creates a component that displays the device form factor. Use the MAUI abstraction layer for native apps and provide an implementation for the web app.
 
-In the Razor class library (RCL), create an `Interfaces` folder and add file named `IFormFactor.cs` with the following code.
+In the Razor class library (RCL), an `Interfaces` folder contains an `IFormFactor` interface.
 
 `Interfaces/IFormFactor.cs`:
 
 :::code language="csharp" source="~/../blazor-samples/8.0/MauiBlazorWeb/MauiBlazorWeb.Shared/Interfaces/IFormFactor.cs":::
 
-In the RCL's `Components` folder, add the following `DeviceFormFactor` component.
+The following `DeviceFormFactor` component is present in the RCL's `Components` folder:
 
 `Components/Pages/DeviceFormFactor.razor`:
 
 :::code language="razor" source="~/../blazor-samples/8.0/MauiBlazorWeb/MauiBlazorWeb.Shared/Components/Pages/DeviceFormFactor.razor":::
 
-In the RCL, add an entry for the `DeviceFormFactor` component to the navigation menu.
+In the RCL, an entry for the `DeviceFormFactor` component is part of the navigation menu in the `NavMenu` component.
```

Comment:
In the new solution template, the form factor UI is on the home page. There isn't a DeviceFormFactor.razor page and there is no change to the NavMenu.